### DISCUSSION TITLE
Activity history tile

### DISCUSSION THREAD

**Discussion by TLeafTO** [jlai/fitness-dashboard#41] [Ideas]
I'd love to see the option to add a "history"  or an "activity" tile to the main page. At present I have to go to history/activities to see my GPS map.
Just a small thing.
Thanks!

**Comment by jlai** [comment]
I've added a tile (`Edit grid` -> `Add tile` -> `Activities`) that lists the activities for the day.

Unlike the old dashboard, it only shows the current day's activities (instead of recent activities); that's needed to cut down on the number of API requests it makes by default (due to the hourly rate limit).

Probably in the future I'll add something to view more activities from the dialog that opens when you click the info icon.

**Reply by TLeafTO** [reply]
Thank you! BTW I continue to promote your Fitness Dashboard in the Fitbit Community!
